xla/pjrt/gpu/se_gpu_pjrt_client.cc

```diff
@@ -1030,6 +1031,23 @@ absl::StatusOr<DeviceTopologyPair> BuildDistributedDevices(
         TF_RET_CHECK(it->second != nullptr);
         local_device = std::move(it->second);
         gpu_device_ids[device_proto.local_device_ordinal()] = global_device_id;
+        // Assign some descriptive names for profiling tools.
```

Comment:
This function is already massive. Maybe we could extract the profiler logic into a subroutine?

Reply:
Sure, done in ae594fa862.